src/project-css/resources/views/admin/schema.blade.php

```diff
@@ -52,8 +52,9 @@
                 <!-- Show the data type to choose from -->
                 <div class="form-group{{$errors->has('col-'.$key.'-data') ? ' has-error' : ''}}">
                   <select id="col-{{$key}}-data" type="text" class="form-control" name="col-{{$key}}-data" required autofocus>
```

Comment:
Select option has no label, this is an accessibility error. 
needs a <label for="col-{{$key}}-data"> Data Type: </lable>